README.md

```diff
@@ -27,7 +27,57 @@ These are the primary software versions used in production, and you should be ab
   - `brew doctor`
 
 
-### **2. Python**  
+### **2. (Mac) Install or update other dependencies:**
```

Comment:
Why did we change the order of the python and mac install?  Is that required?

Reply:
Actually, I had to do the mac installs before I even installed the python ones since they had some dependencies. Thats why I interchanged them